src/js/settings.js

```diff
@@ -117,5 +117,7 @@ function(L) {
     'fillColor': '#fcd96c'
   };
 
+  settings.cors = ('withCredentials' in new XMLHttpRequest());
```

Comment:
`withCredentials` is not necessarily an own-property of `XMLHttpRequest` objects. This is a feature detection, to see if the browser's XHR objects support CORS. So we don't care about the value, we just want to know that those objects have that property (whether they came from the prototype chain or not). We could theoretically check something like `(new XMLHttpRequest()).withCredentials !== undefined`, since that property should either be `false`, `true`, or not exist. But since an object can have a property with value `undefined`, the `in` operator really provides the direct answer we care about.